src/Application/Auth/Token.php

```diff
@@ -21,18 +24,21 @@ class Token
      * @param string $personId UUID for a person
      * @return string Signed JWS
      */
-    public static function create(string $personId): string
+    public static function create(string $personId, bool $complete): string
     {
+        $durationInDays = $complete ? static::VALID_DAYS_PASSWORD_AUTH : static::VALID_DAYS_USER_CREATION;
+
         /**
          * @var array $claims
          * @link https://tools.ietf.org/html/rfc7519 has info on the standard keys like `exp`
          */
         $claims = [
             'iss' => getenv('BASE_URI'),
             'iat' => time(),
-            'exp' => time() + (15 * 86400), // Expire in 15 days
+            'exp' => time() + ($durationInDays * 86400),
             'sub' => [
                 'person_id' => $personId,
+                'complete' => $complete, // A token allows setting a password *or* full Person use; not both.
```

Comment:
Could you pls elaborate on this comment? Not sure what you mean by 'full person use'